projects/packages/my-jetpack/_inc/state/resolvers.js

```diff
@@ -129,6 +130,19 @@ const myJetpackResolvers = {
 				dispatch.setAvailableLicensesIsFetching( false );
 			}
 		},
+
+	getProductData: () => {
+		return async ( { dispatch } ) => {
+			dispatch.setProductDataIsFetching( true );
+
+			try {
+				dispatch.setProductData( await apiFetch( { path: REST_API_SITE_PRODUCT_DATA_ENDPOINT } ) );
+				dispatch.setProductDataIsFetching( false );
+			} catch ( error ) {
+				dispatch.setProductDataIsFetching( false );
```

Comment:
You can use try / catch / finally. and in finally run `dispatch.setProductDataIsFetching( false );`. But it's very optional nitpick

Reply:
I think that seems unnecessary in this case personally. I do it used elsewhere in this file but for the most part it is not. I'll leave it off for now 😄